sidebarsLearn.js

```diff
@@ -182,7 +182,28 @@ module.exports = {
             type: 'category',
             label: 'Use cases',
             items: [
-              'use-case-airflow-dbt',
+              'use-case-airflow-dbt'
+            ]
+          },
+          {
+            type: 'category',
+            label: 'How-to Connections',
```

Comment:
```suggestion
            label: 'Connections',
```
I would just call this section 'Connections'